src/main/kotlin/net/leanix/vsm/gitlab/broker/connector/adapter/feign/FeignDoraProvider.kt

```diff
@@ -0,0 +1,34 @@
+package net.leanix.vsm.gitlab.broker.connector.adapter.feign
+
+import net.leanix.vsm.githubbroker.connector.domain.Dora
+import net.leanix.vsm.gitlab.broker.connector.adapter.feign.data.DoraRequest
+import net.leanix.vsm.gitlab.broker.connector.domain.DoraProvider
+import net.leanix.vsm.gitlab.broker.connector.domain.GitLabAssignment
+import net.leanix.vsm.gitlab.broker.connector.domain.Repository
+import net.leanix.vsm.gitlab.broker.shared.Constants.GITLAB_ENTERPRISE_CONNECTOR
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Component
+
+@Component
+class FeignDoraProvider(
+    private val vsmClient: VsmClient
+) : DoraProvider {
+
+    private val logger = LoggerFactory.getLogger(FeignDoraProvider::class.java)
```

Comment:
Let's try to use the new log library:
private val logger = KotlinLogging.logger {}